Source/WTF/wtf/text/WTFString.h

```diff
@@ -65,7 +65,9 @@ class String final {
     // Construct a string with Latin-1 data.
     WTF_EXPORT_PRIVATE String(std::span<const LChar> characters);
     WTF_EXPORT_PRIVATE String(std::span<const char> characters);
-    ALWAYS_INLINE static String fromLatin1(const char* characters) { return String { characters }; }
+    WTF_EXPORT_PRIVATE String(NullTerminated characters);
```

Comment:
This is a new implicitly Latin-1 character set constructor but the old code had the one most like this, which took a `const char*`, intentionally private. But I must admit we did add a public `std::span<const char>` one that is implicitly Latin-1.